src/styles/components/_SearchBar.scss

```diff
@@ -0,0 +1,131 @@
+.StyleGuide_SearchContainer {
+	background: $hm-primary;
+	width: 100%;
+	clear: both;
+	padding: $base-line-height $gutter-width;
+}
+
+.SearchBar {
+	@extend .util-Clearfix;
+	clear: both;
+	position: relative;
+}
+
+.SearchBar_Container {
+	@extend .util-Clearfix;
+	position: relative;
+
+	@media #{ $mq-md-up } {
+		width: 50%;
+		float: right;
+		transition: width ease-in-out .1s;
+	}
+}
+
+.SearchBar_Label {
+	position: absolute;
+	clip: rect( 1px, 1px, 1px, 1px );
+}
+
+.SearchBar .SearchBar_Field {
```

Comment:
Could do `input.SearchBar_Field` instead? Maybe add a comment to explain the need for specificity.